src/components/tasks/SuggestionBox/SuggestionBox.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+import SuggestionList from './SuggestionList';
+import Loading from './Loading';
+import UserNotFound from './UserNotFound';
+
+type Props = {
+    onClickName: (userName: string) => void;
+    loading: boolean;
+    suggestions: GithubInfo[];
+};
+
+const SuggestionBox: FC<Props> = ({ onClickName, suggestions, loading }) => {
+    let renderComponent = (
+        <SuggestionList suggestions={suggestions} onClickName={onClickName} />
+    );
+
+    if (loading) {
+        renderComponent = <Loading />;
+    } else if (suggestions.length === 0) {
+        renderComponent = <UserNotFound />;
+    }
+
+    return <>{renderComponent}</>;
```

Comment:
Could we remove the fragment here the renderComponent does not contain multiple tags right? fragment is used when we want to return multiple HTML elements/tags to consolidate them into one.